test/multiapi/packages/multiapi-test/.npmignore

```diff
@@ -0,0 +1,35 @@
+#git
```

Comment:
This file should be removed.

Reply:
This is autogenerated.

Reply:
Oh.. it shouldn't generate it after this PR https://github.com/Azure/autorest.typescript/pull/344. Could you try to delete the file and regenerate? If it's still generated, probably I missed something in my PR.